src/components/v2v3/V2V3Project/V2V3Project.tsx

```diff
@@ -53,13 +53,16 @@ export function V2V3Project() {
     fundingCycle,
     isPreviewMode,
     projectOwnerAddress,
+    fundingCycleMetadata,
     handle,
   } = useContext(V2V3ProjectContext)
   const { projectId, cv } = useContext(ProjectMetadataContext)
   const {
     nftRewards: { rewardTiers: nftRewardTiers },
   } = useContext(NftRewardsContext)
 
+  console.info('data: ', fundingCycleMetadata?.dataSource)
```

Comment:
nit

Reply:
if u just use console.log in dev, then the linter will catch it :)

Reply:
Ah nice good call, thanks